app/src/main/java/a3/com/convo/adapters/FriendAdapter.java

```diff
@@ -17,23 +17,39 @@
 import com.facebook.AccessToken;
 import com.facebook.GraphRequest;
 import com.facebook.GraphResponse;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseFile;
+import com.parse.ParseQuery;
 import com.parse.ParseUser;
 
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
 import a3.com.convo.GlideApp;
 import a3.com.convo.R;
 
 public class FriendAdapter extends RecyclerView.Adapter<FriendAdapter.ViewHolder> {
-    private int selectedPos = RecyclerView.NO_POSITION;
-    private ArrayList<String> myFriends;
     Context context;
+    // Your friends usernames
+    private ArrayList<String> myFriends;
+    // Selected position on RV/selected friend(s)
+    private int selectedPos = RecyclerView.NO_POSITION;
     String selectedFriend;
-
+    // For display
+    ParseUser currentFriend;
+    String profPic, name;
+    // Parse columns
+    static String username = "username";
```

Comment:
Lets make them `private static final` these strings are not used outside this class. also, we usually use this format for constants.

`private static final String CONTEXT = "context"`